packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -145,7 +145,7 @@ export const getStyles: IStyleFunction<IDropdownStyleProps, IDropdownStyles> = p
           ],
           ['&:focus .' + globalClassnames.title]: [
             !disabled && rootHoverFocusActiveSelectorNeutralDarkMixin,
-            { borderColor: palette.themePrimary },
+            { borderColor: semanticColors.inputFocusBorderAlt },
```

Comment:
this feels a bit random, but cannot find anything more appropriate

Reply:
is this a border around the focused menu item? should probably just use focusBorder

Reply:
`focusBorder` is not `themePrimary` though...

Reply:
double check with design, we usually aren't as strict on design standards for accessibility-only UI, it just needs to meet contrast ratios

Reply:
reverted this change for now